assignment/week_3/RecommendedMenu/src/components/Recommendation.jsx

```diff
@@ -0,0 +1,85 @@
+import styled from "styled-components";
+import { MENU } from "../constants/menu";
+import { useEffect, useState } from "react";
+import ShowSelectedCategory from "./ShowSelectedCategory";
+
+const Recommendation = (props) => {
+  const category = props.category;
```

Comment:
```suggestion
const Recommendation = ({category}) => {
```
마찬가지로 요롷게 받아버리면 좋겠죠 ?-?